src/hooks/use-site-usage.ts

```diff
@@ -19,10 +24,11 @@ export function useSiteUsage() {
 		}
 		setIsLoading( true );
 		try {
-			const response: { site_count: number; site_limit: number } = await client.req.get( {
-				path: '/jurassic-ninja/usage',
-				apiNamespace: 'wpcom/v2',
-			} );
+			const response: { site_count: number; site_limit: number; sites: UsageSite[] } =
```

Comment:
We could infer site_count from the number of sites.

Reply:
I know, however I think not to change API to keep consistency with users that haven't updated studio, and still have an older version